contracts/distribution/ZNSPriceOracle.sol

```diff
@@ -77,6 +77,8 @@ contract ZNSPriceOracle is AccessControlled, Initializable, IZNSPriceOracle {
 
         fee = getRegistrationFee(domainPrice);
         totalPrice = domainPrice + fee;
+
+        return (totalPrice, domainPrice, fee);
```

Comment:
let's do one or the other method. and not half both. either remove this line or specify naming locally. imo, it was already fine

Reply:
removed here